code/modules/scrap/oldificator.dm

```diff
@@ -0,0 +1,250 @@
+/obj/proc/make_old()
+	color = pick("#996633", "#663300", "#666666")
+	light_color = color
+	name = pick("old ", "expired ", "dirty ") + initial(name)
```

Comment:
More adjectives!

Rusty
ancient
weathered
beaten
broken
tarnished
frayed
etc, get a thesaurus out, make this a big list

Reply:
the list of adjectives isn't as broad, because they are used for any object, so for say "rusty plastic spoon" would make no sense.